doc/May-2022-Upgrade.md

```diff
@@ -0,0 +1,634 @@
+## Upgrading from an old version of kobo-docker (before May 2022)
+
+Current versions of kobo-docker require PostgreSQL 14, MongoDB 5 and Redis 6
+
+
+If you are running a version of kobo-docker that was last updated prior to
+May 2022 (i.e. commit TBC or older),
+you need to upgrade your databases prior to using the current version of
```

Comment:
Replace "prior to" with "before" (it's already been used in the sentence and sounds a bit off)